package/scripts/ambari_helpers.py

```diff
@@ -43,7 +43,20 @@ def cdap_config(name=None):
        content=InlineTemplate(params.cdap_env_sh_template)
   )
 
-  cleanup_opts()
+  if name == 'auth':
```

Comment:
There's no `Cask standard` but the Ambari code uses 2 spaces... so I did the same.

https://github.com/apache/ambari/blob/trunk/ambari-server/src/main/resources/common-services/HDFS/2.1.0.2.0/package/scripts/datanode.py